support-e2e/tests/landingPages.test.ts

```diff
@@ -16,7 +16,10 @@ test.describe("Paper product page", () => {
     const domain = new URL(pageUrl).hostname;
     await setTestCookies(context, firstName(), domain);
     await page.goto(pageUrl);
-    await page.locator("id=qa-paper-subscriptions").isVisible();
+    const pageRendered = await page
+      .locator("id=qa-paper-subscriptions")
+      .isVisible();
+    expect(pageRendered).toBeTruthy();
```

Comment:
An alternative way I've seen this written is
```suggestion
    expect(page.locator("id=qa-paper-subscriptions")).toBeVisible();
```